### ISSUE TITLE
Have option where query window is always open rather than have to hit key

### ISSUE THREAD

**Issue by jlitz** [Flow-Launcher/Flow.Launcher#1460]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**
always hitting key to bring up  search

**Additional context**
just have a box enabling alway open with no need to hit key  to open

**Comment by jlitz** [comment]
the query window disappears  after a search.   It seems to disappear if i have the 'focus is lost'  box unchecked or checked.  Maybe I am missing something.    Is it possible to have the querry box always visible?

**Comment by taooceros** [comment]
>  the query window disappears  after a search.   It seems to disappear if i have the 'focus is lost'  box unchecked or checked.  Maybe I am missing something.    Is it possible to have the querry box always visible?

Oh sorry i think you are right. Currently we didn't have a good way to do it.

**Comment by jjw24** [comment]
> > the query window disappears  after a search.   It seems to disappear if i have the 'focus is lost'  box unchecked or checked.  Maybe I am missing something.    Is it possible to have the querry box always visible?
> 
> Oh sorry i think you are right. Currently we didn't have a good way to do it.

The hide Flow Launcher when focus is lost is for this exact scenario when the box is unchecked. What do you mean? 

@jlitz so the query window still hides even if you have the option unchecked?

**Comment by jlitz** [comment]
yes

**Comment by onesounds** [comment]
> yes

Do you want the window to be maintained even after executing(enter) the query item?

**Comment by jlitz** [comment]
I think that may be a good addition.  You may have to have an option to clear the inquery after the search so the query box is empty.  Just my thoughts.  Great program.   Someone said Apple Macs have a similar  search feature tho I do not know.

**Comment by jjw24** [comment]
I am pretty sure when the option is unchecked the query window will not hide even after executing the query item, can someone else please confirm.

**Comment by onesounds** [comment]
> I am pretty sure when the option is unchecked the query window will not hide even after executing the query item, can someone else please confirm.

No, the query box disappears. This request is a function that we do not have.

**Comment by jjw24** [comment]
Ohh, I think this is because the plugin result overrode it with hidden=true

**Comment by WalterWolf49** [comment]
Any plans for this? Would be very appreciated to just have a search bar somewhere in the corner of a monitor always available...

**Comment by jjw24** [comment]
Hi there, not at the moment. Feel free if anyone would like to take this on, just assign yourself to this issue to let others know it's in progress.

**Comment by EnigmaSolved** [comment]
Though I don't know that I would use this often, there are times where I could see this being really useful. I would recommend that the UI have a little Pin/Unpinned icon in the corner (perhaps top-right?) to indicate the current status and for toggling the state.
